.travis.yml

```diff
@@ -27,28 +26,27 @@ install:
 - pip install --user tox
 - nohup npm run httpbin &
 - if [ "$CXX" = "g++" ]; then export CXX="g++-4.9" CC="gcc-4.9"; fi
+- gcc -v
 - if ! git clone --depth 1 https://github.com/ni-kismet/emscripten-sdk.git; then cd emscripten-sdk && git remote update -p && git merge --ff-only origin/master && cd ..; fi
 - emscripten-sdk/emsdk activate --build=Release sdk-master-64bit
-- source emscripten-sdk/emsdk_env.sh
 - cat ~/.emscripten
-- emcc -v
-- gcc -v
+- bash -c 'source emscripten-sdk/emsdk_env.sh && emcc -v'
 before_script:
 - while ! nc -q 1 127.0.0.1 64526 </dev/null; do sleep 2; done
 - curl http://127.0.0.1:64526/get?show_env=1
 - make lint
 - npm run lint
-- make
-- make vjs BUILD=debug
-- make vjs BUILD=profile
+- make native
 - make unittest
+- bash -c 'source emscripten-sdk/emsdk_env.sh && make vjs'
```

Comment:
Why is this necessary?  Was emsdk being directly in the environment interfering with other commands?

Reply:
yep, nailed it
> Runs emsdk_env commands as separate shell instances to avoid polluting build environments (was replacing node in PATH with an older and incompatible version)